services/web/server/src/simcore_service_webserver/director_v2_core.py

```diff
@@ -426,6 +426,8 @@ async def retrieve(
     )
     body = dict(port_keys=port_keys)
 
+    # sometimes the director-v2 cannot be reached causing the service to fail
+    # retrying 3 times before giving up for good
```

Comment:
hmm I think there is another problem here.
and I don't think that retrying is the way to fix it, that just mean that that call is going to take ages no? Can you write down that case for the maintenance issues on Friday?

Reply:
Yes will take a note of that for Friday maintenance cases.